user-operator/src/test/java/io/strimzi/operator/user/operator/KafkaUserOperatorMockTest.java

```diff
@@ -282,7 +280,7 @@ public void testCreateTlsUserWithACLsDisabled() throws ExecutionException, Inter
                     .endQuotas()
                 .endSpec()
                 .build();
-        KafkaUserOperator op = new KafkaUserOperator(ResourceUtils.createUserOperatorConfig(Map.of(), false, false, "32", null), client, mockCertManager, scramOps, quotasOps, aclOps, EXECUTOR);
+        KafkaUserOperator op = new KafkaUserOperator(ResourceUtils.createUserOperatorConfig(Map.of(), false, false, "32", ""), client, mockCertManager, scramOps, quotasOps, aclOps, EXECUTOR);
```

Comment:
Why did we moved from `null` to `""`?

Reply:
I did it because earlier I was using a single constructor while defining the ConfigParameter. I think with the new refactoring I can revert this change back

Reply:
I think with `null` it better captures the difference between not set and set to n empty string which is not always the same. This applies to the default values of the configuration options as well.